plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1410,13 +1410,18 @@ blockly.block.dimension_left=If player leaves dimension %1
 blockly.block.item_consumed=If %1 is consumed
 blockly.block.item_damaged=If %3 is damaged, damage between %1 and %2
 blockly.block.item_in_inventory=If player has from %1 to %2 %3 in inventory
+blockly.block.lightning_strike=If player is struck by lightning
```

Comment:
The wiki says Triggers when a lightning bolt disappears from the world, only for players within a 256 block radius of the lightning bolt.` It doesn't mention the fact a player is stroke by a lightning bolt.

Also tthis one is too specific. It doesn't add customization (with entity/player customization for example, which I already planned to do later when I'm done with item conditions), so every mod/datapack adding it would have the same behaviour. Please remove it or make big changes like adding entity struck by it .

If you keep it, please add `bystanders` in the format I used for potiuon effects and enchantments, This will make it more customizable while waiting for me to add the player condition part to advancements.